src/containers/Ledger/actions.js

```diff
@@ -23,10 +23,26 @@ export const loadLedger = (identifier, rippledSocket) => (dispatch) => {
   })
 
   return getLedger(identifier, rippledSocket)
-    .then((data) => {
-      dispatch({ type: actionTypes.FINISH_LOADING_FULL_LEDGER })
-      dispatch({ type: actionTypes.LOADING_FULL_LEDGER_SUCCESS, data })
-    })
+    .then((data) =>
+      getLedgerData(identifier, rippledSocket)
```

Comment:
This is only going to retrieve the first page of ledger_data.  There will be many many pages.  This is why there should be a micro service that caches this info.

Reply:
Oh yeah, that is true. I can refactor this.

Reply:
> This is only going to retrieve the first page of ledger_data. There will be many many pages. This is why there should be a micro service that caches this info.

I recommend you only do the first milestone of the bounty in this PR, and leave the second milestone for a separate PR (or multiple PRs).